### DISCUSSION TITLE
How to enable docstring snippet completion for a filetype

### DISCUSSION THREAD

**Discussion by mdmjmsa** [saghen/blink.cmp#734] [Q&A]
[Here](https://github.com/rafamadriz/friendly-snippets/wiki#extending-via-lazynvim) is an example of how to enable the friendly-snippets `doc` snippets for a certain filetype, e.g. enabling `cppdoc` for `cpp`. How to achieve this in `blink.cmp`? I'm currently not having any `doc` snippets using the default config, and the documentation doesn't seem to mention anything about this.

**Comment by idnsunset** [comment]
Same problem, I've set `opts.sources.providers.snippets.opts.extended_filetypes` but still get no completion.
